### DISCUSSION TITLE
其实我是神，v我50我给你无上神力

### DISCUSSION THREAD

**Discussion by wozaofale** [share121/inter-knot#368] [其他]
![Screenshot_20240805_203630](https://github.com/user-attachments/assets/013bfb94-2f58-42ef-ae5a-9184c185485a)

**Comment by wozaofale** [comment]
无敌黄泉如何呢
![Screenshot_20240809_214439_com miHoYo hkrpg](https://github.com/user-attachments/assets/abb50c7a-74e8-4af1-8b00-3e061175df5c)

**Comment by mizukanain** [comment]
我勒个佩恩第七道[崩坏星穹铁道]啊
